lib/broadway_kafka/producer.ex

```diff
@@ -162,6 +162,26 @@ defmodule BroadwayKafka.Producer do
   will reassign all its topic/partition to other available consumers, including any Broadway
   producer subscribed to the same topic.
 
+  ## Consumer Rebalancing
+
+  If you are dealing with more than one partitions and have more than one node consuming the data, you'll
+  find yourself in a situation which one node joining the consumer group causes other consumers to go to rebalancing mode
+  and depends on amount messages you are processing, it might take a long time until they go into the balance state.
+  In this case try to adjust:
+
+  * Producer's `concurrency` value: By increasing this value, you are dividing the partitions among more processors.
+
+  * ':rebalance_timeout_seconds` option: By increasing this value, you are telling Kafka's consumer group coordinator
+  to wait longer time for other member of a consumer group before triggering rebalance.
+
+
+  There is no one config fit all. Keep adjusting until it works for you.
```

Comment:
When do I know it "worked"? Is it when the warning no longer happens?

Reply:
Hmm, that's good point!

Until your nodes don't go to the endless round of rebalancing. But measuring and monitoring that is a bit hard at this moment. 

We do it by monitoring the consumer group lag. If it goes up in a period of time, we know something is off. Ideally brod has a telemetry that when a rebalancing triggered, it push it to the telemetry. I'm working on a PR for brod to add telemetry.

As for the documentation in this module, I'd say until you stop seeing continues rebalancing in a short period of time in your log. Not sure how to put it in proper sentence 😬 

We could also wait for my PR in brod, after telemetry is added. we can point out to that telemetry event.

Reply:
I would write exactly what you wrote here. I can give a pass later to help tidy it up if you are not confident. :)

Reply:
I'll end up removing two suggestions and only left explanation about `rebalance_timeout_seconds` settings. 

We saw increasing the producer concurrency helps but since #93 is merged. is not necessary anymore.